py/DREAM/Settings/Equations/DistributionFunction.py

```diff
@@ -67,6 +67,15 @@ def __init__(self, settings, name, grid,
         self.T0  = rT0
         self.rT0 = T0
 
+        # Ripple parameters
```

Comment:
would be nice to have the settings in RadialGrid instead, so that you don't have to specify these settings separately for f_hot and f_re